docs/how-to-guides/smtp-configuration.md

```diff
@@ -0,0 +1,9 @@
+To configure Discourse's SMTP you'll have to set the following configuration options with the appropriate values for your SMTP server by running `juju config [charm_name] [configuration]=[value]`.
+
+Set `smtp_address`to the SMTP server IP or hostname, `smtp_port` for the SMTP sever port if different from the default and `smtp_domain` to set the sender domain address.
+
+If authentication is needed, `smtp_authentication` will need to be set to the appropriate authentication method. The credencials can be set with `smtp_username`and `smtp_password`.
```

Comment:
I think we should include possible values for `smtp_authentication`. Also, s/credencials/credentials/

Reply:
Fixed